### PR TITLE
Vet Clinic Database - Create animals table

### PR DESCRIPTION
**Pull request by omarsalem7**
Hello 🌷,

This is my PR for the Vet Clinic Database by using PostgreSQL. In this PR, I achieved the following using SQL queries:

- [x] Created a database named vet_clinic
- [x] Created a table named animals with corresponding columns upon the requirements
- [x] Inserted data of 4 animals with the corresponding name, date_of_birth, weight, neutered status, and escape attempts
- [x] Wrote SQL queries to answer questions based on stated requirements.

Thank you so much for reviewing my PR. 🏅
Don't forget to give a ⭐️ to support me 😉

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,29 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+-- Find all animals whose name ends in "mon".
+SELECT * FROM animals WHERE name LIKE '%mon%';
```
Comment (SajjadAhmad14) [inline queries.sql]: Please add a query that can find all animals whose name ends with `mon` not start and ends with `mon`.

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,29 @@
 /*Queries that provide answers to the questions from all projects.*/
```
Comment (SajjadAhmad14) [inline queries.sql:2]: Please add the project title to the README file. A professional README should have a project title, description, and a set of instructions to run the project locally.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Omar, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by omarsalem7** [comment]
Hi @SajjadAhmad14 
I made the changes that you are stating, you can check them now.
Have a nice day

**Review by julie-ify — APPROVED** [review]
Hi @omarsalem7, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**